Validations.h

```diff
@@ -0,0 +1,16 @@
+#ifndef ASSIGNMENT_1_VALIDATIONS_H
+#define ASSIGNMENT_1_VALIDATIONS_H
+
+#include <iostream>
+#include <cfenv>
+
+using namespace std;
+
+class Validations {
```

Comment:
I think, just as we did with menu options, we do not need to define this as a class. We do not have a constructor for this, and I think you will be able to access these options if you just include the header files in options